CHANGELOG.md

```diff
@@ -58,6 +58,16 @@ This was added in pull requests:
 - [#2818: Add support for localisation via data-* attributes to Accordion component](https://github.com/alphagov/govuk-frontend/pull/2818)
 - [#2826: Add support for localisation via JavaScript configuration to Accordion component](https://github.com/alphagov/govuk-frontend/pull/2826)
 
+#### Suppress deprecation warnings
+
+You can now suppress warnings from deprecations within GOV.UK Frontend by updating the `$govuk-suppressed-warnings` map in sass. Every deprecation warning will now include a warning "key" which you can use in the following code, placed at the root of your sass project:
+
+```
+$govuk-suppressed-warnings: append($govuk-suppressed-warnings, $key);
```

Comment:
I don't think this is what we want people to do for setting the initial state of the list (especially as I'm not sure you can append to a non-existent list).

In user land this could just be:

```scss
$govuk-suppressed-warnings: (
  legacy-palette,
  legacy-font
);
```